WordPress/Classes/ViewRelated/Tools/Time Zone/TimeZoneSelectorViewController.swift

```diff
@@ -65,6 +65,18 @@ struct TimeZoneSelectorViewModel: Observable {
         })
     }
 
+    private var timeZoneFormatter: TimeZoneFormatter?
+
+    init(state: State, selectedValue: String?, filter: String?) {
+        self.state = state
+        self.selectedValue = selectedValue
+        self.filter = filter
+
+        if FeatureFlag.timeZoneSuggester.enabled {
```

Comment:
This works well, thanks for the patience @sla8c !
I was thinking that, we might initialize `TimeZoneFormatter` anyway, and then use the feature flag only to decide what type of row we want to return. This so we don't over-complicate the code with unnecessary `guard let timeZoneFormatter = timeZoneFormatter else`; also, since we are working on a feature branch, if we spot performance issues of some sort, we can still fix them before we merge into `develop`

Reply:
Sounds good @Gio2018  - that's done!